ccd-definition/CaseEventToFields.json

```diff
@@ -350,6 +420,18 @@
     "PageColumnNumber": 1,
     "FieldShowCondition": "respondentsMigrated=\"No\""
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "amendRespondents",
+    "CaseFieldID": "respondents",
```

Comment:
this is not needed, let's just give permissions for the new respondents1

Reply:
Removed.